content/account-admin/_index.md

```diff
@@ -37,13 +37,24 @@ A list of all active [integrations][1] in your account.
 Home to all of your [notifications][2], divided into the following categories: Email, OpsGenie, PagerDuty, SNS, Slack, and Webhook.
 
 ### Users
-Create & assume sub-accounts from the users panel. There are three types of users:
-
-- **Account Owners**: Can create & assume all user types.
-- **Admins**: Can create & assume admin + read-only users.
-- **Read-only Users**: Cannot create or assume other users.
-
-Account Owners are the only user group that can view billing details for CloudWisdom.
+Create & assume sub-accounts from the users panel. There are four types of users:
```

Comment:
Sub-accounts can't be assumed from the users panel. That's the Sub-accounts page.